ios/brave-ios/Sources/Growth/DAU.swift

```diff
@@ -251,7 +252,7 @@ public class DAU {
       return true
     }
 
-    let daysThatMustPassToSkipDtoi = AppConstants.buildChannel == .nightly ? 2 : 30
+    let daysThatMustPassToSkipDtoi = 30
```

Comment:
@iccub any idea what this 2 day period on nightly was for? The unit tests even tested against 30 days so I removed it

Reply:
easier for QA to test. Fine to remove by now